tools/frr-reload.py

```diff
@@ -295,6 +309,12 @@ def load_from_file(self, filename):
             # Compress duplicate whitespaces
             line = " ".join(line.split())
 
+            # remove 'vrf <vrf_name>' from 'interface <x> vrf <vrf_name>'
+            if line.startswith("interface ") and "vrf" in line:
+                intf_vrf = re.search("interface (\S+) vrf (\S+)", line)
```

Comment:
do we have to split this re text processing in two like this? if the "startswith" test succeeds, could we just pass the current line into the normalizer function, and let it decide what to do and what to return?

Reply:
@mjstapp I have re organized the code and ensure the normalizer function takes `line` as argument and returns `new line`. I have retained the startswith is to optimize look up and if and only if the startswith passes then perform actual regex will provide the vrf line to cut.